react/features/invite/components/InfoDialogButton.web.js

```diff
@@ -179,6 +199,23 @@ class InfoDialogButton extends Component<Props, State> {
         this.setState({ showDialog: false });
     }
 
+    _onClickOverflowMenuButton: () => void;
+
+    /**
+     * Opens the Info dialog.
+     *
+     * @returns {void}
+     */
+    _onClickOverflowMenuButton() {
+        const { _dialIn, _liveStreamViewURL } = this.props;
+
+        this.props.dispatch(openDialog(InfoDialog, {
+            dialIn: _dialIn,
+            liveStreamViewURL: _liveStreamViewURL,
+            isInlineDialog: false
+        }));
```

Comment:
I can't add a password from within the dialog. Pressing enter dismisses the dialog.